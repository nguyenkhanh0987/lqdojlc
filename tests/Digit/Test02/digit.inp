100
38901 51783
34627 74671
73602 82536
44108 85099
14495 66590
37250 58785
45102 53620
84915 90640
38918 52526
19730 22656
79626 88240
28184 35049
63012 78823
72723 98611
6757 67047
84827 99346
72295 81163
49433 98170
76916 87506
24874 44188
39168 71034
44165 52053
70931 81505
9707 80629
1946 95673
64222 90903
4728 82029
63173 75329
24304 76497
20208 22211
66591 73836
25570 68628
66109 71240
57946 78735
93225 96621
16410 60624
36730 56246
51707 82037
61620 75239
21628 51325
43094 79843
53999 93238
65276 70226
75609 89970
46799 76335
15551 44140
8709 62897
67638 80641
2659 98302
62558 64147
43137 87966
20224 73511
46092 77677
70180 98418
82643 95006
24467 97925
65513 66842
11948 90422
95161 98880
67670 88578
77869 80743
59739 86760
33027 84099
83289 89307
55650 96731
1782 67805
41541 61656
19226 48659
21126 80537
14411 97245
84173 97913
9166 76259
17736 57466
77146 91394
26867 40203
50118 93562
93322 93807
41217 90084
76054 77111
48063 91660
93587 94094
63329 81121
87783 89894
39734 55700
6433 32009
10968 59985
31179 59481
20719 80786
20522 79430
19191 34525
2273 52769
96207 99140
75395 93993
8654 27738
55590 87077
92381 99839
19296 70918
49814 84392
40604 64020
48856 91778