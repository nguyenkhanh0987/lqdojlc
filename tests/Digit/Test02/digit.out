1438
3824
479
3073
6722
2446
962
330
1466
252
224
851
1012
1015
8292
551
492
2663
501
2797
2913
973
505
8285
10632
1464
9731
731
5526
121
471
4968
501
1430
54
6085
2175
2052
754
4099
3095
2346
543
764
2359
4437
7462
836
10621
43
3261
5755
2569
1020
370
6310
89
8418
142
1077
256
1313
4479
305
2257
9415
1985
4137
6193
8130
440
8118
5317
676
2240
2438
0
3529
80
2876
5
1134
197
1414
4783
6953
3291
6212
6170
2231
8145
139
773
3431
1952
248
5964
2127
2058
2697